src/libraries/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -71,6 +71,7 @@ public static partial class Http
             public static readonly Uri Http2RemoteDeflateServer = new Uri("https://" + Http2Host + "/" + DeflateHandler);
             public static readonly Uri Http2RemoteGZipServer = new Uri("https://" + Http2Host + "/" + GZipHandler);
             public static Uri RemoteLoopServer => new Uri("ws://" + RemoteLoopHost + "/" + RemoteLoopHandler);
+            public static Uri RemoteFQDNServer => new Uri("https://" + Http2NoPushHost + "./");
```

Comment:
Is this because `RemoteEchoServer` would fail?

Reply:
Yes, `RemoteEchoServer` handled by Azure and returned "400 Bad Request" that we discussed in the issue https://github.com/dotnet/runtime/issues/57792#issuecomment-988092671
I may add a comment here.

Reply:
Yes, `RemoteEchoServer` handled by Azure and returned "400 Bad Request" that we discussed in the issue https://github.com/dotnet/runtime/issues/57792#issuecomment-988092671
I may add a comment here.

Reply:
Moved into SSL tests